schemas/candidate-answers.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "https://kalkulacka.ceskodigital.net/schemas/candidate-answers.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "A representation of answers for given candidate to questions",
+  "type": "object",
+  "required": ["id", "candidate_id", "questions"],
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "UUID"
+    },
+    "candidate_id": {
+      "type": "string",
+      "description": "Candidate's UUID"
+    },
+    "questions": {
+      "type": "array",
+      "description": "Candidates' answers on questions",
```

Comment:
Kandidati (stranky) maji moznost i delsi odpovedi / vysvetleni. Coz ve finalnim prehledu vysledku pujde rozkliknout (jestli jsem to pochopil spravne).

Reply:
Oki, upravim :)